eras/allegra/impl/src/Cardano/Ledger/Allegra/TxAuxData.hs

```diff
@@ -107,6 +108,8 @@ newtype AllegraTxAuxData era = AuxiliaryDataWithBytes (MemoBytes AllegraTxAuxDat
   deriving (Generic)
   deriving newtype (Eq, ToCBOR, SafeToHash)
 
+instance ToExpr (AllegraTxAuxData era)
```

Comment:
Just curious, is this because it was missing, or is it relevant to implementing `EqRaw` ?

Reply:
It is not relevant for `EqRaw`, but just like `EqRaw` it is relevant for testing. Ultimately we want to add `ToExpr` instances for all of our types, since that improves failure reporting significantly.

Reply:
Makes sense, thanks!